src/sentry/static/sentry/app/components/banner.tsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import InlineSvg from 'app/components/inlineSvg';
+
+import styled from 'react-emotion';
+import theme from 'app/utils/theme';
+import space from 'app/styles/space';
+import {t} from 'app/locale';
+import localStorage from 'app/utils/localStorage';
+
+import spaceBg from '../../images/background-space.svg';
+
+type Props = {
+  title?: string;
+  subtitle?: string;
+  dismissable?: boolean;
+};
+
+class Banner extends React.Component<Props> {
+  static defaultProps: Partial<Props> = {
+    dismissable: true,
+  };
+
+  handleClick() {
```

Comment:
@doralchan Looks like you're on the right track. For this particular component to be re-usable; we can make this component to be more declarative by passing two props:

1. an `onClickDismiss` handler function that'll be called whenever the "x" (dismiss) button is clicked
2. a `hidden` prop that'll control whether the banner is shown or be hidden.

Then we can move the `localStorage` and friends to the `EventsV2` component within the `src/sentry/static/sentry/app/views/eventsV2/index.tsx` file.

Reply:
I moved local storage to the events v2 page!